.github/workflows/verify.yml

```
@@ -44,7 +44,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.20', '1.21']
+        version: ['1.20', '1.21', '>=1.22.0-rc.1']
```

After #4334 this is the only reference to Go version in this PR.

We can fix it to this specific rc1:
```suggestion
        version: ['1.20', '1.21', '1.22.0-rc.1']
```
but it is probably better to have it flexible, testing with future release candidates until Go 1.22 is released and only then change it to the usual:
```suggestion
        version: ['1.20', '1.21', '1.22']
```

---

If this semantics is supported `>=1.22.0-rc.1`, then it seems fine to me to write it this way